autoload/SpaceVim/layers/lang/typescript.vim

```diff
@@ -59,10 +59,26 @@ function! s:on_ft() abort
           \ 'call SpaceVim#lsp#rename()', 'rename symbol', 1)
   else
     if has('nvim')
+      call SpaceVim#mapping#space#langSPC('nnoremap', ['l', 'c'], 'TSTypeDef',
```

Comment:
What is the different between `SPC l c` and `SPC l g`? and for jump to definition, I recommand to use `g d` instead. in SpaceVim you can use `SpaceVim#mapping#gd#add` to register an `g d` function. and you can check the python layer to see how does it works.

Reply:
SPC l c - navigates to symbol type definition
SPC l g - navigates to symbol definition

Reply:
so, can we replace SPC  l g with  g d, which  is same as other language layer .